en/.spelling

```diff
@@ -0,0 +1,46 @@
+>
+Choreo
+API
+APIs
+COVID-19
+cInlineImage-full 
+microservices
+OpenAPI
+lifecycle
+OAuth 2.0
+developer-centric
+timeframe
+https
+integrations
+cInlineImage-half
+choreo-connectors-list
```

Comment:
Any specific reason to have this?
I guess we need to capitalize the C when we use the term Choreo in all occurrences.

Reply:
Resolving this based on the chat we had regarding this at the team meeting.